Readme.md

```diff
@@ -7,6 +7,7 @@
 ![GitHub forks](https://img.shields.io/github/forks/qxf2/qxf2-page-object-model)
 ![GitHub repo size](https://img.shields.io/github/repo-size/qxf2/qxf2-page-object-model)
 ![GitHub last commit](https://img.shields.io/github/last-commit/qxf2/qxf2-page-object-model)
+![Codacy Badge](https://app.codacy.com/project/badge/Grade/c330930eaad64b9cabb62fee2a84fe69)
```

Comment:
@Archana-GS I see code quality as B. Is this what we expect here?

Reply:
No @avinash010 , after merge all the branches the grade will change from 'B' to 'A'